packages/synapse-react-client/src/components/ChallengeSubmission/AvailableEvaluationQueueList.test.tsx

```diff
@@ -51,53 +60,61 @@ describe('AvailableEvaluationQueueList', () => {
   })
 
   it('handles 0 available evaluations', () => {
-    const { selectInput, staticListHeading } = setUp(defaultProps)
+    const { selectInput, showStaticListButton } = setUp(defaultProps)
 
     screen.getByText('No evaluations found')
     expect(selectInput).toBeNull()
-    expect(staticListHeading).toBeNull()
+    expect(showStaticListButton).toBeNull()
   })
 
   it('handles 1 available evaluation', () => {
-    const { selectInput, staticListHeading } = setUp({
+    const { selectInput, showStaticListButton, staticListHeading } = setUp({
       ...defaultProps,
       evaluations: [mockEvaluationQueue],
     })
 
     screen.getByText(mockEvaluationQueue.name!)
     expect(selectInput).toBeNull()
+    expect(showStaticListButton).toBeNull()
     expect(staticListHeading).toBeNull()
 
     expect(onChangeSelectedEvaluation).toHaveBeenCalledTimes(1)
     expect(onChangeSelectedEvaluation).toHaveBeenCalledWith(mockEvaluationQueue)
   })
 
   it('handles 1 available evaluation that is not selectable', () => {
-    const { selectInput, staticListHeading } = setUp({
+    const { selectInput, showStaticListButton, staticListHeading } = setUp({
       ...defaultProps,
       evaluations: [mockEvaluationQueue],
       isSelectable: false,
     })
 
     screen.getByText(mockEvaluationQueue.name!)
     expect(staticListHeading).toBeNull()
+    expect(showStaticListButton).toBeNull()
     expect(selectInput).toBeNull()
     expect(onChangeSelectedEvaluation).not.toHaveBeenCalled()
   })
 
   it('handles >1 available evaluations', async () => {
-    const { selectInput, staticListHeading, user } = setUp({
-      ...defaultProps,
-      evaluations: [mockEvaluationQueue, secondEvaluation],
-    })
-
-    await waitFor(() => {
-      expect(staticListHeading).not.toBeNull()
-    })
-    screen.getByText(mockEvaluationQueue.name!)
-    screen.getByText(secondEvaluation.name)
+    const { selectInput, showStaticListButton, staticListHeading, user } =
+      setUp({
+        ...defaultProps,
+        evaluations: [mockEvaluationQueue, secondEvaluation],
+      })
 
     expect(selectInput).not.toBeNull()
+    expect(showStaticListButton).not.toBeNull()
+    expect(showStaticListButton?.textContent).toContain('Show')
+    expect(staticListHeading).not.toBeVisible()
+
+    await user.click(showStaticListButton!)
+
+    expect(showStaticListButton?.textContent).toContain('Hide')
+    expect(staticListHeading).toBeVisible()
+    expect(screen.getByText(mockEvaluationQueue.name!)).toBeVisible()
+    expect(screen.getByText(secondEvaluation.name)).toBeVisible()
```

Comment:
Show the list of available evaluation queues -- 

https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/f64e4d4b-9714-41e9-a8e1-82d9dd577b0c